README.md

```diff
@@ -4,28 +4,34 @@
 
 💬 **#language-tools** on our [Discord Server](https://discord.gg/vue)
 
+## Example Projects
+
+- [Basic Example](./examples/basic/) A basic example project that highlights some aspects of the Vue language tools. This example is a good starting point for new users.
+- [[Todo] nvim](./examples/with-nvim/) If you are using Neovim, feel free to contribute an example project to help others.
```

Comment:
Why would there be a editor-specific example here? The editor configuration is a separate concern IMO.

All in all I'm not convinced I see the point of those examples.

Reply:
Editor-specific examples can be used to highlight certain intricacies of different environments. I'm not saying it should be focused on specific editors only, but there are important points in that regard, e.g.

* explain how the hybrid version setting works in vscode
* nvim config files for a good editing experience
* vscode debugging and launcher profiles
* how to configure the project's typescript version in contrast to the typescript version of the vscode workbench

IDE unspecific:
* How to configure global components
* How to setup type declaration in libraries vs Vue applications
* How to setup generic components and what do you need to be aware of in regards to the language tools?
* How to implement different tsconfig setups with project references etc.
* How to configure your nuxt/element-plus/(?) libraries to work well with vscode

etc.

Reply:
IMO editor specific quirks should be documented separately in text or delegated to other projects that take care of that. There is not that much you can show within a project regarding editor configuration.

As for your other (non-editor) points, I suppose some might make sense but it might be encroaching on being typescript/vue territory rather than something that should be documented within a vue language server.